radon/tests/test_raw.py

```diff
@@ -187,7 +187,7 @@ def fib(n):
          """
          if n <= 1: return 1  # otherwise it will melt the cpu
          return fib(n - 2) + fib(n - 1)
-     ''', (6, 9, 10, 2, 3, 1, 1)),
+     ''', (6, 9, 10, 2, 3, 2, 1)),
```

Comment:
I believe that '2' is the correct value for number of blank lines in this code.  I think previously the line in the multiline string had been excluded.